src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/SymbolMatcherTests.cs

```diff
@@ -997,12 +997,14 @@ public void Method_ParameterNullableChange()
             var source0 = @"
 class C
 {
-    void M(string s) { }
+    string c;
+    ref string M(string s, (string a, string b) tuple) => ref c;
 }";
             var source1 = @"
 class C
 {
-    void M(string? s) { }
+    string c;
+    ref string? M(string? s, (string? a, string? b) tuple) => ref c;
```

Comment:
We should also test `dynamic?` - so one of these parameters could by `dynamic?`.
Also I'd add more combinations of nullable to non-nullable and nullable to nullable. (That swap some of the `?` between source0 and source1).